nixos/modules/services/networking/networkmanager.nix

```diff
@@ -327,9 +327,9 @@ in {
             type = mkOption {
               type = types.enum (attrNames dispatcherTypesSubdirMap);
               default = "basic";
-              description = ''
+              description = lib.mdDoc ''
                 Dispatcher hook type. Look up the hooks described at
-                <link xlink:href="https://developer.gnome.org/NetworkManager/stable/NetworkManager.html">https://developer.gnome.org/NetworkManager/stable/NetworkManager.html</link>
+                [https://developer.gnome.org/NetworkManager/stable/NetworkManager.html](https://developer.gnome.org/NetworkManager/stable/NetworkManager.html)
```

Comment:
Why did this not convert to URL link? Also need to update the link to https://networkmanager.dev/docs/api/latest/NetworkManager.html.

Reply:
Oh, I vaguely remember some previous PR of yours doing that but maybe you did it manually there.